lib/stitch_fix/log_weasel/middleware.rb

```diff
@@ -10,18 +14,21 @@ def initialize(app, options = {})
     # Future: Maybe add X-Amzn-Trace-Id request header for tracing through load balancer
     # (http://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-request-tracing.html)
     def call(env)
-      x_request_id = env.fetch("HTTP_X_REQUEST_ID", nil)
+      x_correlation_id = env.fetch(REQUEST_ID_KEY,
```

Comment:
Perhaps a more Ruby way to write this would be:
`x_correlation_id = env.fetch(REQUEST_ID_KEY, nil) || env.fetch(CORRELATION_ID_KEY, nil)`

Reply:
I've updated it to be the ruby way. I do note that seems to stutter and makes me push 5 things onto the stack instead of 3, FWIW.